examples/Google.Cloud.Functions.Examples.FSharpUntypedEventFunction/Function.fs

```diff
@@ -33,7 +33,15 @@ type Function() =
             printfn "Type: %s" cloudEvent.Type
             printfn "Subject: %s" cloudEvent.Subject
             printfn "DataSchema: %A" cloudEvent.DataSchema
-            printfn "DataContentType: %A" cloudEvent.DataContentType
-            printfn "SpecVersion: %A" cloudEvent.SpecVersion
+            printfn "DataContentType: %O" cloudEvent.DataContentType
+            printfn "Time: %s" (match Option.ofNullable cloudEvent.Time with
+                                | Some time -> time.ToUniversalTime().ToString "yyyy-MM-dd'T'HH:mm:ss.fff'Z'"
+                                | None -> "")
+            printfn "SpecVersion: %O" cloudEvent.SpecVersion
             printfn "Data: %A" cloudEvent.Data
+
+            // In this example, we don't need to perform any asynchronous operations, so we
+            // just return a completed Task to conform to the interface.
             Task.CompletedTask
+
+let x = Nullable 10
```

Comment:
Was this intentional? My guess is you were checking something else...

Reply:
Doh!!! That was my just testing out if you could pattern match on Nullables directly (you can't).

Reply:
Just checking - are you pushing a new commit to remove this? I think we're ready to release when you do.

Reply:
Yep, I am.

Reply:
Pushed.